pushnotifications/src/main/java/com/pusher/pushnotifications/reporting/FCMMessageReceiver.kt

```diff
@@ -28,9 +29,21 @@ class FCMMessageReceiver : WakefulBroadcastReceiver() {
         val pusherData = gson.fromJson(pusherDataJson, PusherMetadata::class.java)
         log.i("Got a valid pusher message.")
 
+        if (context == null) {
+          log.e("Failed to get device ID (no context) - Skipping delivery tracking.")
```

Comment:
I'd change the log to a warning and not an error, on the grounds that the application will be 100% ok if it reaches this scenario.